docs/index.asciidoc

```diff
@@ -20,26 +20,11 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
-This input will read events from a Kafka topic. It uses the 0.10 version of
-the consumer API provided by Kafka to read messages from the broker.
-
-Here's a compatibility matrix that shows the Kafka client versions that are compatible with each combination
-of Logstash and the Kafka input plugin: 
-
-[options="header"]
-|==========================================================
-|Kafka Client Version |Logstash Version |Plugin Version |Why?
-|0.8       |2.0.0 - 2.x.x   |<3.0.0 |Legacy, 0.8 is still popular 
-|0.9       |2.0.0 - 2.3.x   | 3.x.x |Works with the old Ruby Event API (`event['product']['price'] = 10`)  
-|0.9       |2.4.x - 5.x.x   | 4.x.x |Works with the new getter/setter APIs (`event.set('[product][price]', 10)`)
-|0.10.0.x  |2.4.x - 5.x.x   | 5.x.x |Not compatible with the <= 0.9 broker
-|0.10.1.x  |2.4.x - 5.x.x   | 6.x.x |
-|0.11.0.0  |2.4.x - 5.x.x   | 6.x.x |Not compatible with the <= 0.9 broker
-|==========================================================
-
-NOTE: We recommended that you use matching Kafka client and broker versions. During upgrades, you should
-upgrade brokers before clients because brokers target backwards compatibility. For example, the 0.9 broker
-is compatible with both the 0.8 consumer and 0.9 consumer APIs, but not the other way around.
+This input will read events from a Kafka topic.
+
+This plugin uses Kafka Client 0.11.0. For broker compatibility, see the official https://cwiki.apache.org/confluence/display/KAFKA/Compatibility+Matrix[Kafka compatibility reference].
```

Comment:
Should this be a call-out/note thing instead of a paragraph here?

Reply:
I think it's good as a paragraph.

Reply:
++ @dedemorton perhaps we can have a "Broker Compatibility" header/title here so its not missed. Thoughts?

Reply:
Also, for the kafka client, can we use 0.11 instead of 0.11.0? I'm not sure what 0.11.0 is, since their versioning is w.x.y.z.

Reply:
I don't know if that's more or less confusing. 0.11 is less precise than 0.11.0. I'd lean towards more precise and say 0.11.0.0 -- what's the goal of 0.11?